### ISSUE TITLE
Update Docker recipe: we now have official images

### ISSUE THREAD

**Issue by oberstet** [crossbario/crossbardocs#61]
From the mailing list, Jonas has had some troubles with Crossbar.io and Docket, which he finally solved. We should test and adjust [the docs](http://crossbar.io/docs/Installation-on-Docker/) ..

```
I have updated the stackoverflow. here my solution :

I work with Docker in order to have a clean environment.
The Dockerfile here : http://crossbar.io/docs/Installation-on-Docker/ seem broken :

    ImportError: No module named setuptools_ext
   
    ----------------------------------------
    Cleaning up...
    Command python setup.py egg_info failed with error code 1 in /tmp/pip-build-VfPnRU/pynacl
    Storing debug log for failure in /root/.pip/pip.log
    The command '/bin/sh -c pip install crossbar[all]' returned a non-zero code: 1

it seem solved with :

    RUN pip install --upgrade cffi

Before RUN pip install crossbar[all]

With this Environment, my problem are solved :)
Don't know why i get this error before, but it's work.

Many thanks to all here and to indexerror, the "french python stackoverflow" :)

http://indexerror.net/3380/crossbar-0-12-1-wsgi-error-no-module-named-django?show=3415




P.S.

Here the clean Dockerfile i use :

    # to build this container:
    # docker build -t crossbar .
    FROM debian:jessie
   
    RUN apt-get update && apt-get -y install build-essential libssl-dev libffi-dev libreadline-dev libbz2-dev libsqlite3-dev libncurses5-dev python-pip python-dev
    RUN pip install --upgrade cffi
    RUN pip install crossbar[all]


2016-02-12 12:52 GMT+01:00 Jonas Turbeaux <jturbeaux@gmail.com>:

    Thanks Tobias, i will try it.
    I'm confused that i'm the only one with this error... Very strange cause i work with a clean environment through docker.
    I see that your exemple use the 0.13 version of crossbar. You get the same result with the 0.12.1 i guess ?
    Many thanks again for your work. It's amazing , i expected this kind of tools since a long time. Good luck :)

    2016-02-12 12:24 GMT+01:00 Tobias Oberstein <tobias.oberstein@gmail.com>:

        Hi Jonas,

        funny, just that minute I added a WSGI example:

        https://github.com/crossbario/crossbarexamples/tree/master/wsgi

        Works for me with CB trunk (and should work with release also, with slight config file mods).

        --

        Rgd Django:

        The https://github.com/crossbario/crossbarexamples/tree/master/django/realtimemonitor for the Django/Crossbar.io tutorial http://crossbar.io/docs/Adding-Real-Time-to-Django-Applications/ works for me out of the box.


        So .. mmh. Can't reproduce your issue =(

        Cheers,
        /Tobias


        Am 12.02.2016 um 11:53 schrieb Jonas Turbeaux:

            Thanks Tobias, no problem ! Take your time, i'm not in a hurry :) I
            Still look for my side, others say that they are unable to reproduce the
            error with flask. I'm just puzzled :)
            Many thanks again :)


            2016-02-12 0:35 GMT+01:00 Tobias Oberstein <tobias.oberstein@gmail.com
            <mailto:tobias.oberstein@gmail.com>>:

                Jonas,

                I'll look tomorrow. Promise;) Need to get some sleep ..

                Cheers,
                /Tobias

                Am 09.02.2016 um 13:43 schrieb Jonas Turbeaux:

                    I get an error while i launch crossbar 0.12.1 that I did not
                    have with
                    the version 0.11.
                    I opened a topic on Stackoverflow :

                    I
                    http://stackoverflow.com/questions/35290280/crossbar-0-12-1-no-module-named-django

                    Many Thanks for your work !
```

**Comment by Nasjoe** [comment]
Here's another Dockerfile Who Works well with me :

```
FROM ubuntu
ENV APPNAME="monappli"

ADD requirements.txt /tmp/

RUN apt-get update
RUN apt-get install -y gcc build-essential python-dev python2.7-dev libxslt1-dev libssl-dev libxml2 libxml2-dev libffi-dev libreadline-dev libbz2-dev libsqlite3-dev libncurses5-dev python-mysqldb python-pip

RUN cd /tmp/ && pip install -r requirements.txt
RUN pip install -U crossbar[all]

WORKDIR $APPNAME

CMD cd / && cd $APPNAME && python manage.py makemigrations && python manage.py migrate && crossbar start
```

Thanks.

**Comment by oberstet** [comment]
We now have official Docker images: https://groups.google.com/forum/#!topic/crossbario/8Qzex8rxV1E

So I am renaming this issue, since actually, the docs needs fixing now ..

**Comment by goeddea** [comment]
Closed via 06b8a488e3251ec3371dd3b2c3063c0a30af3ea5 (but commit cites wrong issue number).
